renv.lock

```diff
@@ -946,7 +946,7 @@
     },
     "knitr": {
       "Package": "knitr",
-      "Version": "1.49",
+      "Version": "1.48",
```

Comment:
Is this is on purpose, or the renv.lock should not be ipdated?

Reply:
no idea. that is just the version I had on my computer

Reply:
OK, we have tested with this renv.lock, so let is so. But better not to change this the next time if it is not necessary: the tests done so far were done with the old renv.lock.

Reply:
I mean I have not changed it actively...